src/components/grid-aware/Footer/SubscriptionBlock/SubscriptionBlock.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Button from "../../../inline/Button";
+import { InputText, InputTextPropType } from "../../../inline/InputText";
+
+import s from "./SubscriptionBlock.module.css";
+
+const SubscriptionBlock = ({ inputText }) => {
+  return (
+    <div className={s.bleedWrapper}>
+      <div className={s.subscriptionBlock}>
+        <div className={s.subscribeContainer}>
+          <div className={s.subscribeMessage}>Subscribe to get updates</div>
```

Comment:
To match our other components, should this class just be named `title`?

Reply:
ok